tests/dotnet/UnitTests/FileHelpers.cs

```diff
@@ -0,0 +1,27 @@
+using System.IO;
+
+#nullable enable
+
+namespace Xamarin.Tests {
+	public static class FileHelpers {
+		public static void CopyDirectory (string sourceDir, string destinationDir)
+		{
+			if (!Directory.Exists (sourceDir))
+				throw new DirectoryNotFoundException ($"Unable to copy source dir, it was not found: {sourceDir}");
+
+			if (!Directory.Exists (destinationDir))
+				Directory.CreateDirectory (destinationDir);
+
+			foreach (var file in Directory.GetFiles (sourceDir)) {
+				var destFile = Path.Combine (destinationDir, Path.GetFileName (file));
+				File.Copy (file, destFile, true);
```

Comment:
FileInfo already has a copy method: https://learn.microsoft.com/en-us/dotnet/api/system.io.fileinfo.copyto?view=net-8.0#system-io-fileinfo-copyto(system-string)  No need to call the static one passing the source, the code already knows it.